src/wheel/bdist_wheel.py

```diff
@@ -286,7 +421,9 @@ def run(self):
 
         # Add to 'Distribution.dist_files' so that the "upload" command works
         getattr(self.distribution, 'dist_files', []).append(
-            ('bdist_wheel', get_python_version(), wheel_path))
+            ('bdist_wheel',
+             '.'.join(list(tags.interpreter_version())),  # like 3.7
```

Comment:
The `list()` seems unnecessary.

Reply:
Is `tags.interpreter_version` ever any different from the Python version?
This will also react badly to Python 3.10.

Reply:
fixing via sys.version_info